src/js/components/FileInput/FileInput.js

```diff
@@ -455,101 +455,125 @@ const FileInput = forwardRef(
           )}
           {files.length > 0 &&
             files.length <= aggregateThreshold &&
-            files.map((file, index) => (
-              <Box
-                key={file.name}
-                justify="between"
-                direction="row"
-                align="center"
-              >
-                {renderFile ? (
-                  renderFile(file)
-                ) : (
-                  <Box
-                    {...theme.fileInput.label}
-                    gap="xsmall"
-                    align="center"
-                    direction="row"
-                  >
-                    {((maxSize && file.size > maxSize) ||
-                      (max && index >= max)) && <CircleAlert />}
-                    <Label
-                      weight={
-                        theme.global.input.weight ||
-                        theme.global.input.font.weight
-                      }
-                      truncate
-                      {...passThemeFlag}
+            files.map((file, index) => {
+              let removeIconA11yTitle = '';
+              if (maxSize && file.size > maxSize) {
+                removeIconA11yTitle = format({
+                  id: 'fileInput.removeMaxFileSize',
+                  messages,
+                  values: {
+                    fileName: file.name,
+                    maxSize: formatBytes(maxSize),
+                  },
+                });
+              } else if (max && index >= max) {
+                removeIconA11yTitle = format({
+                  id: 'fileInput.removeMaxFileCount',
+                  messages,
+                  values: {
+                    fileName: file.name,
+                    maxNumber: max,
+                  },
+                });
+              } else {
+                removeIconA11yTitle = `${format({
+                  id: 'fileInput.remove',
+                  messages,
+                })} file: ${file.name}`;
+              }
```

Comment:
We can simplify this. Will require adding a file name param to the fileInput.remove message. I know it was pre-existing, but when internationalizing, you should never concatenate parts of a message together (instead use substitution vars.)
```suggestion
              let messageId = 'fileInput.remove';
              if (maxSize && file.size > maxSize) {
                messageId = 'fileInput.removeMaxFileSize';
              }
              else if (max && index >= max) {
                messageId = 'fileInput.removeMaxFileCount';
              }
```
Then change the point where it puts in in prop to:
```jsx
                format({
                  id: messageId,
                  messages,
                  values: {
                    fileName: file.name,
                    maxSize: formatBytes(maxSize),
                    maxNumber: max,
                  },
                });
```